src/lib.rs

```diff
@@ -439,11 +446,12 @@ impl Compiler {
 
     /// Parses and typechecks `expr_str` followed by extracting metadata from the created
     /// expression
-    pub fn extract_metadata(&mut self,
-                            vm: &Thread,
-                            file: &str,
-                            expr_str: &str)
-                            -> Result<(ast::SpannedExpr<ast::TcIdent<Symbol>>, TcType, Metadata)> {
+    pub fn extract_metadata
+        (&mut self,
+         vm: &Thread,
+         file: &str,
+         expr_str: &str)
+         -> Result<(ast::SpannedExpr<ast::TcIdent<Symbol>>, TcType, Metadata)> {
```

Comment:
You may be right. Looked at the default settings for rustfmt and I think this is the default. Not sure why I haven't seen it before.